### PR TITLE
[SPARK-25627][TEST] Reduce test time for ContinuousStressSuite

### PR DESCRIPTION
**Pull request by viirya**
## What changes were proposed in this pull request?

This goes to reduce test time for ContinuousStressSuite - from 8 mins 13 sec to 43 seconds.

The approach taken by this is to reduce the triggers and epochs to wait and to reduce the expected rows accordingly.

## How was this patch tested?

Existing tests.

### CODE REVIEW

**Comment by viirya** [comment]
Not sure if the modified triggers/epochs are enough for this test. cc @zsxwing

**Comment by dongjoon-hyun** [comment]
Retest this please.

File: sql/core/src/test/scala/org/apache/spark/sql/streaming/continuous/ContinuousSuite.scala [hunk 1]
```diff
@@ -259,10 +259,10 @@ class ContinuousStressSuite extends ContinuousSuiteBase {
     testStream(df, useV2Sink = true)(
       StartStream(Trigger.Continuous(2012)),
```
Comment (kiszk) [inline sql/core/src/test/scala/org/apache/spark/sql/streaming/continuous/ContinuousSuite.scala]: just curios: Is this value still `2012`? Can a reduction to `1012` reduce time?

Comment (viirya) [reply sql/core/src/test/scala/org/apache/spark/sql/streaming/continuous/ContinuousSuite.scala]: I think 2012 is safer here as it only increases to epoch 1. Smaller value might make this test flaky.

Comment (kiszk) [reply sql/core/src/test/scala/org/apache/spark/sql/streaming/continuous/ContinuousSuite.scala]: Got it, thanks

**Comment by dongjoon-hyun** [comment]
Hi, @tdas and @zsxwing .
Could you review this PR which aims to reduce the test time greatly?

**Review by srowen — APPROVED** [review]

**Comment by viirya** [comment]
ping @tdas and @zsxwing Can you take a look this? Thanks.

**Comment by HyukjinKwon** [comment]
retest this please

**Comment by srowen** [comment]
Merged to master

**Comment by viirya** [comment]
Thanks @srowen @dongjoon-hyun @kiszk

**Pull request closed without merging**